packages/client/src/modules/defaultRouter/index.web.jsx

```diff
@@ -1,12 +1,36 @@
 import React from 'react';
-import { Switch } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Switch, withRouter } from 'react-router-dom';
 
 import modules from '../';
-
+import PageLayout from './components/PageLayout';
```

Comment:
I think, in order to stick to a common approach it's better to pass it as Feature param with a wildcard.

Reply:
Good idea. I will do it.